docs/app/home/home.component.scss

```diff
@@ -136,8 +136,8 @@ section button {
     display: table;
 }
 
-@media screen and (max-width: 740px)
-{
+
+@include media-breakpoint-down(md) {
```

Comment:
This was a mixin I slipped in to make media queries easier, I got the idea from the bootstrap grid system..

https://getbootstrap.com/docs/4.0/layout/grid/